src/lib.rs

```diff
@@ -11,6 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![cfg_attr(test, deny(warnings))]
```

Comment:
I prefer in `.travis.yml` too.
Most of users use stable rust version, so they may still meet "local tests succeed but CI fails".